src/Business/Camera/Fetching/Commands/GetCamera.cs

```diff
@@ -0,0 +1,16 @@
+namespace Business.Camera.Fetching.Commands
+{
+    using System.Collections.Generic;
+    using Business.Camera.Common.Models;
+    using Business.Camera.Common.Repositories;
+    using Business.Camera.Fetching.Models;
+
+    public class GetCamera : IGetCamera
```

Comment:
Would the name `GetCameras` be more relevant here? Should we rename the interface in this case also?

Reply:
Done